db/migrate/20200421024915_remove_measurments_from_tanks.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveMeasurmentsFromTanks < ActiveRecord::Migration[5.2]
```

Comment:
Probably not important? spelling on `Measurement`.
Relevant if we think code editors will pick up on it and autofill to misspellings.

Reply:
Fixed it, thanks!